src/VacancyRssFeedService/VacancyRssFeedService.csproj

```diff
@@ -231,16 +231,7 @@
     <VisualStudio>
       <FlavorProperties GUID="{349c5851-65df-11da-9384-00065b846f21}">
         <WebProjectProperties>
-          <UseIIS>True</UseIIS>
-          <AutoAssignPort>True</AutoAssignPort>
-          <DevelopmentServerPort>56369</DevelopmentServerPort>
-          <DevelopmentServerVPath>/</DevelopmentServerVPath>
-          <IISUrl>https://localhost:44311/</IISUrl>
-          <NTLMAuthentication>False</NTLMAuthentication>
-          <UseCustomServer>False</UseCustomServer>
-          <CustomServerUrl>
-          </CustomServerUrl>
-          <SaveServerSettingsInUserFile>False</SaveServerSettingsInUserFile>
+          <SaveServerSettingsInUserFile>True</SaveServerSettingsInUserFile>
```

Comment:
Not seen that before, works in some ways as you always have your personal setup correct. Breaks other ways if apps need to have specific ports to work together when running multiple projects / solutions

Reply:
I prefer it as it puts the responsibility onto the dev running the code. They can then run it whichever way they prefer/are set up for